NitroxClient/MonoBehaviours/Gui/Settings/SettingsManager.cs

```diff
@@ -0,0 +1,87 @@
+﻿using NitroxClient.Unity.Helper;
+using NitroxModel.Logger;
+using Steamworks;
+using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.UI;
+
+namespace NitroxClient.MonoBehaviours.Gui.Settings {
+    public class SettingsManager {
+        private static string Name;
+        private static string Key_Chat;
+        private static float Color_R;
+        private static float Color_G;
+        private static float Color_B;
+        public static Image Color_Image;
+
+        public static void SetName(string name) {
+            Name = name;
+        }
+        public static string GetName() {
```

Comment:
All these get and set functions can be converted to properties.

For Name, I prefer an auto property, which is initialized to `SteamFriends.GetPersonaName()` in the constructor.
But you can also add initial validation in a regular property.